core/include/vecmem/containers/impl/jagged_vector_buffer.ipp

```diff
@@ -128,10 +103,18 @@ jagged_vector_buffer<TYPE>::jagged_vector_buffer(
       m_outer_host_memory(::allocate_jagged_buffer_outer_memory<TYPE>(
           sizes.size(),
           (host_access_resource == nullptr ? resource
-                                           : *host_access_resource))),
-      m_inner_memory_size(0u),
-      m_inner_memory(::allocate_jagged_buffer_inner_memory<TYPE>(
-          capacities, resource, true, m_inner_memory_size)) {
+                                           : *host_access_resource))) {
+    using header_t = typename vecmem::data::jagged_vector_buffer<
+        TYPE>::value_type::size_type;
+    // Determine the allocation size.
+    std::size_t total_elements = std::accumulate(
+        capacities.begin(), capacities.end(), static_cast<std::size_t>(0));
+
+    header_t* header_ptr;
+    TYPE* data_ptr;
```

Comment:
This is the static code analyzer talking... These pointers should be explicitly initialised to `nullptr`. Otherwise static analyzers will complain.